int8 stop_reason
---
bool success